### ISSUE TITLE
libfalcon requires tokio `=1.38` and can no longer be updated in Omicron

### ISSUE THREAD

**Issue by iliana** [oxidecomputer/falcon#89]
https://github.com/oxidecomputer/falcon/blob/0f47d13588a2226720645a7b19481872cd3a263e/Cargo.toml#L36

This seems to have changed in https://github.com/oxidecomputer/falcon/pull/84; Omicron is using tokio 1.39. I am trying to incorporate #84 into Omicron to deal with the bare Serialize/Deserialize issue, as part of doing a larger lockfile maintenance PR.

```
error: failed to select a version for `tokio`.
    ... required by package `libfalcon v0.1.0 (https://github.com/oxidecomputer/falcon?rev=0f47d13588a2226720645a7b19481872cd3a263e#0f47d135)`
    ... which satisfies git dependency `libfalcon` of package `omicron-test-utils v0.1.0 (/home/iliana/git/omicron/test-utils)`
    ... which satisfies path dependency `omicron-test-utils` (locked to 0.1.0) of package `cert-dev v0.1.0 (/home/iliana/git/omicron/dev-tools/cert-dev)`
versions that meet the requirements `=1.38` are: 1.38.1, 1.38.0

all possible versions conflict with previously selected packages.

  previously selected package `tokio v1.39.3`
    ... which satisfies dependency `tokio = "^1.39.3"` (locked to 1.39.3) of package `cert-dev v0.1.0 (/home/iliana/git/omicron/dev-tools/cert-dev)`

failed to select a version for `tokio` which could resolve this conflict
```

Was the intent to set a _minimum_ version? In that case the `=` should not be present.

**Comment by rcgoodfellow** [comment]
I pinned it because recent mishaps in tokio completely broke falcon. I'll push an update shortly that moves things to 1.39.3.

**Comment by rcgoodfellow** [comment]
Should be fixed in 10f3bff98d247d1e145a0dde82382cbed3628cdc

**Comment by iliana** [comment]
Yep! Thanks :)
